osu.Game/Configuration/OsuConfigManager.cs

```diff
@@ -175,6 +175,7 @@ protected override void InitialiseDefaults()
             SetDefault(OsuSetting.EditorWaveformOpacity, 0.25f);
 
             SetDefault(OsuSetting.LastProcessedMetadataId, -1);
+            SetDefault(OsuSetting.ComboColourBrightness, 0.7f, 0f, 1f);
```

Comment:
Not sure this should have a minimum of 0. Makes everything pure black which feels a bit unusable.